### ISSUE TITLE
Wrong SQL for cumulative count / cumulative sum

### ISSUE THREAD

**Issue by sbelak** [metabase/metabase#7126]
Original question:
![screen shot 2018-03-15 at 4 22 33 pm](https://user-images.githubusercontent.com/178038/37472761-210f4574-286d-11e8-80a9-eeb530291b39.png)

Turned into SQL:
![screen shot 2018-03-15 at 4 22 04 pm](https://user-images.githubusercontent.com/178038/37472778-278d86d6-286d-11e8-8cfa-3328b8856564.png)

**Comment by camsaul** [comment]
Cumulative count and cumulative sum require post-processing to get the results. I think we should perhaps consider disabling "convert to SQL" for those two since they are not going to give you the same results

**Comment by mattpera** [comment]
Hello,
Is there a way to manually activate cumulative post-processing count / sum on a SQL based question currently? If not, would that be a possible functionality to consider adding?

**Comment by lindsay-stevens** [comment]
@mattpera if your database supports SQL window functions, it's possible to calculate the cumulative totals in the query: https://stackoverflow.com/questions/5698452/count-cumulative-total-in-postgresql e.g.

```sql
select dim, sum(count(*)) over (order by dim) from table group by dim order by dim
```

**Comment by nemanjaglumac** [comment]
Seems to have been fixed on the latest `master` at d35c779976ae6bd79369e9cc6bec8540908f37d2

GUI
![Image](https://github.com/metabase/metabase/assets/31325167/5f91004c-6b64-4a31-8a91-685bed87c3be)

New SQL I get is:
```sql
SELECT
  FLOOR(("PUBLIC"."ORDERS"."TOTAL" / 20.0)) * 20.0 AS "TOTAL",
  COUNT(*) AS "count"
FROM
  "PUBLIC"."ORDERS"
GROUP BY
  FLOOR(("PUBLIC"."ORDERS"."TOTAL" / 20.0)) * 20.0
ORDER BY
  FLOOR(("PUBLIC"."ORDERS"."TOTAL" / 20.0)) * 20.0 ASC
```

![Image](https://github.com/metabase/metabase/assets/31325167/e55ecd0b-02a8-47e6-b03e-ebaf18fc075c)


And for cumulative sum:

GUI
![Image](https://github.com/metabase/metabase/assets/31325167/785b5f00-c2b2-4cb0-b3d7-60bd823f9441)

SQL
![Image](https://github.com/metabase/metabase/assets/31325167/e9549f6d-a199-4eee-8995-d16dbdb928c1)

**Comment by bshepherdson** [comment]
This is mis-attributed to MBQL-Lib - converting questions to SQL is done with an API call in the FE code.

There's nothing here to write tests for; so I'm moving this to No Repro for the bug bash.
